make/testing_flags.mk

```diff
@@ -69,4 +69,4 @@ endif
 # Construct the integration test command with the added build flags.
 ITEST_TAGS := $(DEV_TAGS) $(RPC_TAGS) rpctest $(backend)
 
-ITEST := rm lntest/itest/*.log; date; $(GOTEST) -v ./lntest/itest -tags="$(ITEST_TAGS)" $(TEST_FLAGS) -logoutput -goroutinedump
+ITEST := rm lntest/itest/*.log; rm -r lntest/itest/.minerlogs; rm -r lntest/itest/.backendlogs; date; $(GOTEST) -v ./lntest/itest -tags="$(ITEST_TAGS)" $(TEST_FLAGS) -logoutput -goroutinedump
```

Comment:
is this really needed? Won't they be deleted on shutdown, or alternatively overwritten on next run?

Reply:
I have doubts too. Normally it would be fine until there are `.gz` files, which won't be cleaned if a sequence of itest fails consecutively.
It might be more related to a different PR that focuses on fixing `.gz` files?

Reply:
Yeah, I would make that a separate PR if it is a problem.

Reply:
Now removed.